src/join/composer.jl

```diff
@@ -1243,7 +1365,8 @@ semijoin(df1::AbstractDataFrame, df2::AbstractDataFrame;
          matchmissing::Symbol=:error) =
     _join(df1, df2, on=on, kind=:semi, makeunique=makeunique,
           indicator=nothing, validate=validate,
-          left_rename=identity, right_rename=identity, matchmissing=matchmissing)
+          left_rename=identity, right_rename=identity, matchmissing=matchmissing,
+          order=:left)
```

Comment:
Does this have a cost?

Reply:
no, it is ignored, but I have to pass something so I pass the value that is respected. The same for `antijoin`